Lib/msilib/text.py

```diff
@@ -1,4 +1,5 @@
-import msilib,os;dirname=os.path.dirname(__file__)
```

Comment:
I have doubts about this. `dirname` is not directly used too. But the code in `msilib.Binary.__repr__()` looks suspicious. If it returns a Python code that should be executed in the context of this module, the `msilib` name should be present.